src/filter/facedetect/facedetect.cpp

```diff
@@ -21,6 +21,9 @@
 #include <stdio.h>
 #include <string.h>
 #include <opencv2/opencv.hpp>
+#if CV_MAJOR_VERSION >= 4
+# include <opencv2/imgproc/imgproc_c.h>
+#endif
```

Comment:
I don't like this very much. This should be fixed by switching to the newer C++ API, see #99.

Reply:
if #99 fixes compat withOpenCV4, by all means please go ahead and use that!